Docker.DotNet/Models/HostConfig.cs

```diff
@@ -12,6 +12,9 @@ public class HostConfig
         [DataMember(Name = "Links")]
         public IList<string> Links { get; set; }
 
+        [DataMember(Name = "GroupAdd")]
+        public IList<string> GroupAdd { get; set; }
```

Comment:
I can't seem to find GroupAdd in https://docs.docker.com/engine/reference/api/docker_remote_api_v1.20/ ?